apps/meeting/src/constants/index.ts

```diff
@@ -51,4 +51,4 @@ export const SDK_LOG_LEVELS = {
   'warn': LogLevel.WARN,
   'error': LogLevel.ERROR,
   'off': LogLevel.OFF,
-}
\ No newline at end of file
+};
```

Comment:
nit: newline end of file.

Reply:
> We still have many `any` type in our code, instead of disable eslint for those lines, I would prefer to `disable no-explicit-any` rule in our config.
> 
> ```
> rules: {
>     "@typescript-eslint/no-explicit-any": "off"
>   },
> ```
> 
> Maybe we should also disable `@typescript-eslint/no-empty-function` rule.

I think we should not centralize `no-explicit-any` as it is a bad practice to disable it completely rather the `any` should be avoided if we can in henceforth code changes.

A more better rule would be allow any for the comments describing why it is used and that can be a centralized rule. I did not have time to go over each of them to fix so disabled on usage as of now.